src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -223,8 +224,9 @@ object UserService {
   def genWorkbenchUserId(currentMilli: Long): WorkbenchUserId =
     WorkbenchUserId(genRandom(currentMilli))
 
-  def validateEmailAddress(email: WorkbenchEmail): IO[Unit] = {
+  def validateEmailAddress(email: WorkbenchEmail, domainBlacklist: Seq[String]): IO[Unit] = {
     email.value match {
+      case emailString if domainBlacklist.exists(domain => emailString.endsWith("@" + domain)) => IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"email domain not permitted [${email.value}]")))
```

Comment:
I'm not sure the `qwiklabs-gsuite.net` domain would match this because the emails are of the format:
```
student@goog-test.reseller.gappslabs.co.s-ixlmdsbu.qwiklabs-gsuite.net
```

Reply:
fixed